public/css/icinga/forms.less

```diff
@@ -425,18 +425,12 @@ form.inline {
 }
 
 // Disabled inputs
-
-.icinga-controls .toggle-switch.disabled {
+.icinga-controls input[type="checkbox"]:disabled + .toggle-switch {
```

Comment:
This was previously done with the `disabled` class.

* Why is this removed?
* Who set this to begin with?
* Is it used somewhere still?